website/src/utils/search.ts

```diff
@@ -63,7 +63,13 @@ const getFieldOrColumnVisibilitiesFromQuery = (
         if (field.hideOnSequenceDetailsPage === true) {
             return;
         }
-        visibilities.set(field.name, initiallyVisibleAccessor(field) === true);
+
+        let fieldName = field.name;
+
+        if (field.rangeOverlapSearch) {
+            fieldName = field.rangeOverlapSearch.rangeName;
+        }
+        visibilities.set(fieldName, initiallyVisibleAccessor(field) === true);
```

Comment:
The modifications in this function could be avoided if we force users to uses a `field.name` as the `rangeName`. This can't really be enforced. IMO this can't be avoided because we have to group fields two distinct fields.

Happy for input though if anybody happens to look at the PR already!